Tests/StreamChatTests/Controllers/ChannelController/ChannelController_Tests.swift

```diff
@@ -4182,16 +4182,16 @@ final class ChannelController_Tests: XCTestCase {
         }
         client.setToken(token: .unique(userId: .unique))
 
+        let updater = try XCTUnwrap(env.channelUpdater)
```

Comment:
why was this needed?

Reply:
This was because I needed to move the logic how we fetch the last read message. Before it uses observer, now it fetches from the DB which caused a delay and I needed to change tests to take this into account.